### DISCUSSION TITLE
Home assistant integration?

### DISCUSSION THREAD

**Discussion by ecto1a2003** [gpstar81/GPStar-proton-pack#245] [General]
Iv'e started work on a home assistant integration for this now that the attenuator can connect to wifi, is anyone interested in this?

**Comment by Ecto11** [comment]
i have just connected mine up to the wifi.. and would love to see a function like.. älexa, activate proton pack for display or something

**Comment by ecto1a2003** [comment]
Hi there i kinda started working on this but python is jot my native language and im aure theres a better way to Getthe info into home assistant.

https://github.com/ecto1a2003/GPstar-ProtonPack-HA

**Comment by nomakewan** [comment]
Looks like someone beat y'all to it, actually.

https://www.facebook.com/groups/spenglersworkshop/posts/1403945447249446

**Reply by ecto1a2003** [reply]
Im not sure what im supposed to see here, i dont have a Facebook account

**Reply by nomakewan** [reply]
I just tested in a browser with no Facebook login and the video played just fine (since both the group and reel are public), but in case you're having strange issues with your browser I've attached the video by Evan Perry below.

https://github.com/user-attachments/assets/e9c63e77-9c11-4026-8feb-9b93d8849629

**Reply by ecto1a2003** [reply]
Ah neat, my idea was to expose every feature into my smart home, not just for control but for reporting and triggering of automations as well .
